blog/2016/02/23/introducing-chrome-debugger-for-vs-code.md

```diff
@@ -48,7 +48,9 @@ We believe there’s much to be done for front-end developers, as building for t
 
 We’ve released this extension on [GitHub](https://github.com/Microsoft/vscode-chrome-debug) as an MIT licensed open source project. It’s a work in progress, so see our issues page for known bugs – for example, we have a few issues around supporting all flavors of generated source maps.
 
-We really want your feedback and help to build an even better debugging experience, so if you have any issues or ideas for improvements, feel free to reach out to us on [Twitter](https://twitter.com/EdgeDevTools) or [GitHub](https://github.com/Microsoft/vscode-chrome-debug/).
+For now we are supporting Google Chrome, but we’re also looking at expanding our support to [Microsoft Edge](https://www.microsoft.com/en-us/windows/microsoft-edge) and other browsers. We really want your feedback and help to build an even better debugging experience
```

Comment:
Let's link to http://dev.microsoftedge.com/ instead since this is a developer audience.